mypy/stubtest.py

```diff
@@ -1273,11 +1274,20 @@ def verify_var(
                 isinstance(proper_type, mypy.types.Instance)
                 and proper_type.type.fullname in mypy.types.ELLIPSIS_TYPE_NAMES
             ):
-                should_error = False
+                value_t = stub.info.get("_value_")
+                if value_t is None or value_t.type is None:
+                    should_error = False
+                elif runtime_type is not None and is_subtype_helper(runtime_type, value_t.type):
+                    should_error = False
+                elif runtime_type is not None:
+                    note = " (incompatible '_value_')"
```

Comment:
Is this intentionally making the `runtime_type is None` case error now?

Reply:
I also tried thinking about that, couldn't invent a case where we have found an attribute but failed to interpret its `value`, and decided to be cautious, reporting such "smth weird" cases as incompatibilities. Maybe this should be an `assert`, but I'm afraid to accidentally kill stubtest right before release...